subliminal/providers/itasa.py

```diff
@@ -0,0 +1,390 @@
+
+        if int(root.data.count.text) == 0:
+            logger.warning('Subtitles not found')
+            return []
+
+        # Looking for subtitlles in first page
+        for subtitle in root.data.subtitles:
+            if '%dx%02d' % (season, episode) in subtitle.find('name').text.lower():
+
+                logger.debug('Found subtitle id %d - %s - %s',
+                             int(subtitle.id.text),
+                             subtitle.find('name').text,
+                             subtitle.version.text)
+
+                sub = ItaSASubtitle(
+                        int(subtitle.id.text),
+                        subtitle.show_name.text,
+                        season,
+                        episode,
+                        format,
+                        subtitle.find('name').text)
+
+                subtitles.append(sub)
+
+        # Not in the first page of result try next (if any)
+        next = root.find('next')
+        while next.text != '':
+
+            r = self.session.get(next.text, timeout=30)
+            r.raise_for_status()
+            root = ParserBeautifulSoup(r.content, ['lxml-xml', 'lxml', 'html.parser']).find('root')
+
+            logger.info('Loading subtitles page %s', root.data.page.text)
+
+            # Looking for show in following pages
+            for subtitle in root.data.subtitles:
+                if '%dx%02d' % (season, episode) in subtitle.find('name').text.lower():
+
+                    logger.debug('Found subtitle id %d - %s - %s',
+                                 int(subtitle.id.text),
+                                 subtitle.find('name').text,
+                                 subtitle.version.text)
+
+                    sub = ItaSASubtitle(
+                        int(subtitle.id.text),
+                        subtitle.show_name.text,
+                        season,
+                        episode,
+                        subtitles.version.text,
+                        subtitle.find('name').text)
+
+                    subtitles.append(sub)
+
+            next = root.find('next')
+
+        # Dowload the subs found, can be more than one in zip
+        additional_subs = []
+        for sub in subtitles:
+
+            # open the zip
+            content = self._download_zip(sub.sub_id)
+            if not is_zipfile(io.BytesIO(content)):
+                if 'limite di download' in content:
+                    raise TooManyRequests()
+                else:
+                    raise ConfigurationError('Not a zip file: %r' % content)
+
+            with ZipFile(io.BytesIO(content)) as zf:
+                if len(zf.namelist()) > 1:
+
+                    for name in enumerate(zf.namelist()):
+
+                        if name[0] == 0:
+                            # First elemnent
+                            sub.content = fix_line_ending(zf.read(name[1]))
+                            sub.full_data = name[1]
+                        else:
```

Comment:
No, it is realative to same release, just a different version. Sometime is a better resync or commented by translator.